client/src/components/SharedComponents/Signup/signup.css

```diff
@@ -0,0 +1,27 @@
+body {
+  text-align: center;
+}
+
+button.signup {
+  border-radius: 20px;
+  padding: 2%;
+  margin-top: 3%;
+  width: 20%;
+  font-size: 130%;
```

Comment:
Font sizes should never be percentages, either pixels, em or rem. Have a quick research on these.